vite.config.ts

```diff
@@ -1,6 +1,14 @@
 import { sveltekit } from '@sveltejs/kit/vite';
 import { defineConfig } from 'vite';
+import { resolve } from 'path'
 
 export default defineConfig({
-	plugins: [sveltekit()]
+	plugins: [sveltekit()],
+	resolve: {
```

Comment:
I will do some more investigating on this one, I don't think it's supposed to be necessary ($lib, $components, are supposed to work by default and they do for build but not test), so I might have something else configured incorrectly.

Reply:
it does seem strange, yes. The $ prefix was working for me fine.